package.json

```diff
@@ -14,6 +14,7 @@
     "license": "MIT",
     "devDependencies": {
         "@napi-rs/cli": "2.10.0",
+        "@types/node": "*",
```

Comment:
If it is something that should be exported with the package, should be in dependencies

Reply:
I am not entirely sure if `@types/node` should be a dependency or a devDependency.

But since they are only needed for development and shouldn't end up in packaged apps, I assume, that a devDependency is a better fit here...

Reply:
Typescript doesn't export types on the final build. It should be a dependency because when a consumer installs the `steamworks.js` library, it also should have the `@types/node` installed to compile successfully.

Tbh I'm also not entirely sure of this hehe